src/clients/index.ts

```diff
@@ -1,9 +1,10 @@
-import { Apps, Builder, Events, InstanceOptions, IOContext, Logger, Registry, Router, Workspaces } from '@vtex/api'
+import { Apps, Events, InstanceOptions, IOContext, Logger, Registry, Router, Workspaces } from '@vtex/api'
 import { getAccount, getToken, getWorkspace } from '../conf'
 import * as env from '../env'
 import envTimeout from '../timeout'
 import userAgent from '../user-agent'
 import Billing from './billingClient'
+import Builder from './Builder'
```

Comment:
You can delete this line and import the Builder from `@vtex/api` when you move the new client code to the lib.